### PR TITLE
[HNC-474] : Add Citadel reports as a notification for Slack, Github S…

### PR DESCRIPTION
**Pull request by AdityaHitachi**
Add Citadel step tracking in Github summary, slack and Teams.
Also based on the team discussion, the BlackDuck step has been removed from the merge workflow tracking

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -156,33 +156,42 @@ runs:
...
       shell: bash
-      if: ${{ always() && env.BlackDuck_Project_Name }}
+      if: ${{ always() && env.TRIGGER_CITADEL_SCAN }}
       run: |
         icon="boom"
-        [[ "${{ steps.blackduck-scan.outcome }}" == "success" ]] && icon="white_check_mark"
+        [[ "${{ steps.citadel-scan.outcome }}" == "success" ]] && icon="white_check_mark"
 
-        sed -i "s/blackduck_scan=x/blackduck_scan=${icon}/g" $RUNNER_TEMP/icons.properties
+        sed -i "s/citadel_scan=x/citadel_scan=${icon}/g" $RUNNER_TEMP/icons.properties
```
Comment (cardosov) [inline action.yaml:167]: I don't think we want to remove the Black duck step completely.

**Review by cardosov — CHANGES_REQUESTED** [review]

File: action.yaml (hunk 1 above)
Comment (AdityaHitachi) [reply action.yaml:168]: Ok. For now added Blackduck as commented code

Comment (cardosov) [reply action.yaml:168]: But why? If it's not being called, doesn't matter if the code is there. The OWASP step code is also there even if it's not being called.

Comment (AdityaHitachi) [reply action.yaml:168]: yes we keep that code. But for tracking do we still need to track blackduck along with Citadel ??

Comment (cardosov) [reply action.yaml:168]: For what we know at the moment, we only track Citadel.

Comment (AdityaHitachi) [reply action.yaml:168]: OK

Comment (AdityaHitachi) [reply action.yaml:168]: Uncommented the blackduck step

File: action.yaml [hunk 2]
```diff
@@ -156,8 +157,17 @@ runs:
...
+      shell: bash
+      if: ${{ always() && env.TRIGGER_CITADEL_SCAN }}
+      run: |
+        icon="boom"
+        [[ "${{ steps.citadel-scan.outcome }}" == "success" ]] && icon="white_check_mark"
+
+        sed -i "s/citadel_scan=x/citadel_scan=${icon}/g" $RUNNER_TEMP/icons.properties
         
-    # Blackduck section
+    Blackduck section
```
Comment (cardosov) [inline action.yaml:170]: You need to keep this line commented.

**Review by cardosov — CHANGES_REQUESTED** [review]

File: action.yaml (hunk 2 above)
Comment (AdityaHitachi) [reply action.yaml:170]: oh my mistake 😅

**Review by cardosov — APPROVED** [review]

**Review by vganpisetty — APPROVED** [review]

**Pull request merged**